requirements.txt

```diff
@@ -1,7 +1,7 @@
-asyncio==3.4.3
```

Comment:
I would also like to remove the strict == requirements in near-lake-framework (otherwise we can't really move to python 3.12 or beyond).

Reply:
hmm. I used the `==` because in case the package makes an update with breaking changes in the future, it's guaranteed to work with the specified version. But since that syntax won't work with future python versions, then we can change it to `>=`.